students/Osiddiquee/lesson02/print_grid2.py

```diff
@@ -0,0 +1,16 @@
+def print_grid2(x,y):
```

Comment:
Good job! 
A few small things:
- x and y aren't the most descriptive parameter names
- the first two loops of loops can be replaced with using string multiplication
Overall good work!